test/Feature/RangeSpec.hs

```diff
@@ -29,12 +29,12 @@ spec = beforeAll (clearTable "items" >> createItems 15) . afterAll_ (clearTable
             }
 
         it "returns range Content-Range with range/*" $
-          request methodGet "/items?order=id"
+          request methodGet "/items?id=eq.1&order=id"
```

Comment:
What about leaving the old test in place and adding this one with the filter as an additional test rather than replacing the old test? Might as well cover both paths in the code.